syntax = "proto3";
option cc_enable_arenas = true;

package Ydb.Auth;
option java_package = "com.yandex.ydb.auth";

import "ydb/public/api/protos/ydb_operation.proto";

message LoginRequest {
    Ydb.Operations.OperationParams operation_params = 1;
    string user = 2;
    string password = 3;
}

message LoginResponse {
    // After successfull completion must contain LoginResult.
    Ydb.Operations.Operation operation = 1;
}

message LoginResult {
    string token = 1;
}
